app/src/main/java/com/android/sample/ui/authentication/SignUp.kt

```diff
@@ -305,13 +258,7 @@ fun CreateNewAccountButton() {
               Text(
                   modifier = Modifier.testTag("createAccountText"),
                   text = "Create New Account",
-                  style =
-                      TextStyle(
-                          fontSize = 14.sp,
-                          lineHeight = 20.sp,
-                          fontFamily = FontFamily(Font(R.font.roboto)),
-                          fontWeight = FontWeight(500),
-                          letterSpacing = 0.14.sp))
+                  style = TypographyBeatLink.labelLarge)
```

Comment:
Consider utilizing MaterialTheme.typography to further centralize theme control and maintain consistency throughout the application.

```suggestion
                  style = MaterialTheme.typography.labelLarge)
```